package.json

```diff
@@ -197,12 +197,13 @@
     "jest-fail-on-console": "3.1.1",
     "jest-junit": "16.0.0",
     "jest-matcher-utils": "29.3.1",
-    "lerna": "5.5.4",
+    "lerna": "7.4.1",
     "mini-css-extract-plugin": "2.7.6",
     "msw": "1.3.2",
     "mutationobserver-shim": "0.3.7",
     "ngtemplate-loader": "2.1.0",
     "node-notifier": "10.0.1",
+    "nx": "16.10.0",
```

Comment:
nx is a required peer dependency of lerna now?

Reply:
Thanks @joshhunt for questioning this. Seems like it's not required so have removed it. Can't remember why I added it. 🤔